src/sentry/static/sentry/app/views/organizationDiscover/styles.jsx

```diff
@@ -43,6 +43,19 @@ export const ResultTitle = styled(Box)`
   min-width: 70px;
 `;
 
+export const ResultViewButtons = styled(Flex)`
+  @media (max-width: 950px) {
+    display: none;
+  }
+`;
+
+export const ResultViewDropdownButtons = styled('div')`
+  display: none;
+  @media (max-width: 950px) {
+    display: contents;
```

Comment:
Why `display: contents`? I believe this isn't implemented by all our target browsers

Reply:
thanks didn't know that, changing it to `display: block`